src/styles/abstracts/_mixins.scss

```diff
@@ -55,6 +59,84 @@
 	}
 };
 
+/**
+ * Generators
+ */
+
+@mixin generate-properties($_properties, $_value) {
+	@if meta.type-of($_properties) == string {
+		#{$_properties}: $_value;
+	} @else {
+		@each $_prop in $_properties {
+			#{$_prop}: $_value;
+		}
+	}
+};
+
+/**
+ * Utility class generator
+ *
+ * usage:
+	// Value list
+	$align: ('left', 'right', 'center');
+	@include generate-utils('text', 'text-align', $align);
+	// output: u-text-left, u-text-right, u-text-center
+
+	// Value map
+	$spacers: (
+		'8': $spacer-xs,
+		'16': $spacer-sm,
+	);
+	$mx-props: ('margin-left', 'margin-right');
+	@include generate-utils('mx', $mx, $spacers);
```

Comment:
Moet `$mx` hier niet `$mx-props` zijn? Ik weet dat het commentaar is, maar voor de duidelijkheid :)

Reply:
Klop! Will fix